Assets/MRTK/Providers/Oculus/XRSDK/Editor/OculusXRSDKConfigurationChecker.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.MixedReality.Toolkit.XRSDK.Oculus
     [InitializeOnLoad]
     static class OculusXRSDKConfigurationChecker
     {
-        private const string AsmDefFileName = "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef";
+        private const string AsmDefFileName = "MRTK.Oculus.asmdef";
```

Comment:
@RogPodge Is this a script that can be cleaned up a bit after https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8655? It looks like you edited one Oculus config checker there, but not this one

Reply:
yeah we can remove these I'll make a PR